### PR TITLE
REL-4280 Fixes for IGRINS-2 support on obslog

### PR DESCRIPTION
**Pull request by cquiroz**

### CODE REVIEW

File: bundle/edu.gemini.obslog/src/main/java/edu/gemini/obslog/instruments/Igrins2LogSegment.java [hunk 1]
```diff
@@ -17,7 +17,7 @@
 public class Igrins2LogSegment extends InstrumentLogSegment {
     public static final Logger LOG = Logger.getLogger(Igrins2LogSegment.class.getName());
 
-    private static final String NARROW_TYPE = "Igrins2";
+    private static final String NARROW_TYPE = "IGRINS-2";
```
Comment (swalker2m) [inline bundle/edu.gemini.obslog/src/main/java/edu/gemini/obslog/instruments/Igrins2LogSegment.java:20]: Does this have to match the `SpComponentType`?  If so, maybe we should just take it from there:
```java
private static final String NARROW_TYPE = SpComponentType.INSTRUMENT_IGRINS2.narrowType;
```

**Review by swalker2m — APPROVED** [review]
Looks good to me. :+1:

File: bundle/edu.gemini.obslog/src/main/java/edu/gemini/obslog/instruments/Igrins2LogSegment.java (hunk 1 above)
Comment (cquiroz) [reply bundle/edu.gemini.obslog/src/main/java/edu/gemini/obslog/instruments/Igrins2LogSegment.java:20]: Right, let me try that

**Review by swalker2m — APPROVED** [review]
:+1:

**Pull request merged**